classes/feeds.php

```diff
@@ -21,8 +21,14 @@ public static function get_videos(array $params=array(), $user=false)
 
 		$params = array_merge($default_params, $params);
 
-		$url = 'http://gdata.youtube.com/feeds/api/users/'.$user.'/uploads?';
-		$url .= http_build_query($params);
+		$url = 'http://gdata.youtube.com/feeds/api/users/'.$user.'/uploads';
+		// The parameters will generate a feed from YouTube's search index, effectively
+		// restricting the result set to indexed, public videos, rather than returning
+		// a complete list of the user's uploaded videos.
+		// https://developers.google.com/youtube/2.0/developers_guide_protocol_video_feeds#User_Uploaded_Videos
+		if (count($params) > 1) {
```

Comment:
Is this necessary? `$params` has always at least 1 element - it's merged with `$default_params` a few lines above.